certified-connectors/TALXISDataFeed/apiProperties.json

```diff
@@ -0,0 +1,112 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "aad",
+          "clientId": "28d529aa-b85e-4469-9cf3-937bea582555",
```

Comment:
@honzakostejn Please remove the client id value as per Microsoft's security policy. You can replace it with dummy text.
`"clientId": "[[DUMMY]]"`

Reply:
We're using another application registration for the API. This one for connector has only rights to call the API itself. But I'll replace it.